Database/Corrections/sodQuestFixes.lua

```diff
@@ -1674,6 +1679,42 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -400099,
             [questKeys.zoneOrSort] = sortKeys.ROGUE,
         },
+        [90088] = {
+            [questKeys.name] = "First Trial of The Owls",
+            [questKeys.startedBy] = {{210044}},
```

Comment:
```suggestion
            [questKeys.startedBy] = {nil,{410020}},
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5378/commits/bfe0509185f2ffde9c120eac643cc22f9e92d074